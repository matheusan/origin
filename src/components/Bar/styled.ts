import styled from 'styled-components';

export const Bar: JSX.Element = styled.div`
  display: flex;
  top: 0;
  left: 0;
  right: 0;
  height: 80px;

  padding: 24px 56px;
  background-color: ${props => props.theme.colors.neutralWhite};
  align-items: center;

  @media only screen and (max-width: 380px) {
    height: 56px;
  }
`;
