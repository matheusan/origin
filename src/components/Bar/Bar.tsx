import * as React from 'react';

import * as S from './styled';

export interface BarProps {
  children: React.ReactNode;
}

const Bar: React.SFC = ({ children }) => <S.Bar>{children}</S.Bar>;

export default Bar;
