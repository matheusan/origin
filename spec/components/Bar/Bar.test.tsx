import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import Bar from '../../../src/components/Bar';

describe('Bar', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() => {
    wrapper = shallow(<Bar>Bar content</Bar>);
  });

  it('renders', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
