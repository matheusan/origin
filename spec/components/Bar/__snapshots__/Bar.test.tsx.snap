// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar renders 1`] = `
<styled.div>
  Bar content
</styled.div>
`;
